doc/source/whatsnew/v0.25.0.rst

```diff
@@ -254,6 +254,7 @@ Groupby/Resample/Rolling
 - Bug in :func:`pandas.core.groupby.GroupBy.agg` when applying a aggregation function to timezone aware data (:issue:`23683`)
 - Bug in :func:`pandas.core.groupby.GroupBy.first` and :func:`pandas.core.groupby.GroupBy.last` where timezone information would be dropped (:issue:`21603`)
 - Ensured that ordering of outputs in ``groupby`` aggregation functions is consistent across all versions of Python (:issue:`25692`)
+- Bug in :func:`pandas.core.groupby.GroupBy.idxmin` and :func:`pandas.core.groupby.GroupBy.idxmax` where datetime columns would incorrectly return datetime values instead of ints (:issue:`25444`, :issue:`15306`)
```

Comment:
so can you be a bit more clear here. something like: when grouping and asking for the idxmin/max of a datetime column you would not receive the index dtype, rather the dtype of the date column.

just want a reader who is upgrading to have a clue if this bug affects them.

Reply:
whats new entry updated by using your hint (https://github.com/pandas-dev/pandas/pull/25531/commits/28b5ab2fe70420867aed8d09bde27df53e5ef1fd). I tried to generalize it to fit both issues.